### DISCUSSION TITLE
In the search results it is necessary to add something to indicate making a choice

### DISCUSSION THREAD

**Discussion by Maofree** [Oksydan/falcon#392] [Ideas]
### Description

Hi
I've seen that the result pages have not set the order selection
the same problem on the classic theme
in the classic theme instead a text is shown to indicate making a choice

I try to do so into sort-orders.tpl

![1](https://github.com/Oksydan/falcon/assets/653230/80d86ed6-23cd-4089-a340-f5b34fbc4ccc)

otherwise an option that is not the one they are seeing comes up and they understand that they have to make a selection

[sort-orders.zip](https://github.com/Oksydan/falcon/files/14424521/sort-orders.zip)

bye

### Node.js version

v18

### php version

8.1

### OS and it's version

linux

### Browsers

Chrome

### Required module/theme

theme

### Reproduction steps

```bash
1. Go to a search result
2. look at the sort selection
```

**Comment by Oksydan** [comment]
Hi @maofree,

tbh list is always sort in one way. If you didn't chose anything it's still sorted. I don't think it is Falcon or Classic error but Prestashop core itself. It should pass sorting order by default for search results something like `search accuracy(??)` for example.
